### ISSUE TITLE
Formatting breaks element generics

### ISSUE THREAD

**Issue by huuff** [bram209/leptosfmt#156]
Before formatting:

![image](https://github.com/user-attachments/assets/df96ea09-0bdf-41ee-98e4-9799259cdb2b)

After formatting:

![image](https://github.com/user-attachments/assets/cc20e497-565f-48ab-9643-d52188b9c66e)

Note that after formatting, it won't compile anymore.

Sorry to include screenshots, I think it was the best way to show the issue

**Comment by MikeOnTea** [comment]
Yep, and it also removes the generic on the closing tag, which at least leptos 0.7 requires.

**Comment by bram209** [comment]
> Yep, and it also removes the generic on the closing tag, which at least leptos 0.7 requires.

This should be fixed in the latest version of leptosfmt.



> Before formatting:
> 
> ![image](https://github.com/user-attachments/assets/df96ea09-0bdf-41ee-98e4-9799259cdb2b)
> 
> After formatting:
> 
> ![image](https://github.com/user-attachments/assets/cc20e497-565f-48ab-9643-d52188b9c66e)
> 
> Note that after formatting, it won't compile anymore.
> 
> Sorry to include screenshots, I think it was the best way to show the issue

Can you try again in the latest version of leptosfmt as well?

**Comment by MikeOnTea** [comment]
So the missing generic on the closing tag is fixed with 1.33, but the opening tag misformatting remains:
Original:
```rust
		<FormDialog<CreatePlace> open_title="Neuer Ort".to_owned() submit_title="Anlegen".to_owned() on_success=on_add>
			<TextInput id="name".to_owned() title="Name".to_owned() required=true />
		</FormDialog<CreatePlace>>
```
Formatted:
```rust
		<FormDialog<
		CreatePlace,
	> open_title="Neuer Ort".to_owned() submit_title="Anlegen".to_owned() on_success=on_add>
			<TextInput id="name".to_owned() title="Name".to_owned() required=true />
		</FormDialog<CreatePlace>>
```

**Comment by bram209** [comment]
Thanks for checking!
